metron-platform/metron-indexing/src/test/java/org/apache/metron/indexing/dao/InMemoryMetaAlertDao.java

```diff
@@ -200,4 +207,23 @@ public MetaAlertCreateResponse createMetaAlert(MetaAlertCreateRequest request)
     createResponse.setCreated(true);
     return createResponse;
   }
+
+  @Override
+  public boolean addAlertsToMetaAlert(String metaAlertGuid, Collection<String> alertGuids,
+      Collection<String> sensorTypes) throws IOException {
+    return true;
```

Comment:
Given the non-implementation of these, do we not have any integration tests in metron-rest exercising the add/remove alerts calls?

Reply:
MetaAlertControllerIntegrationTest would be the best place to add these integration tests, and I think they should be added to make sure the REST calls line up the way we do everything else.

Reply:
I think @cestella's suggestion is good, I will add some REST integration tests that ensure request/responses are serialized/deserialized correctly.  There are tests included in MetaAlertControllerIntegrationTest that exercise these functions.